tests/namespace_delete.go

```diff
@@ -136,16 +137,16 @@ func (s *namespaceTestSuite) Test_NamespaceDelete_InvalidUTF8() {
 	s.NoError(err)
 
 	descResp, err := s.frontendClient.DescribeNamespace(ctx, &workflowservice.DescribeNamespaceRequest{
-		Namespace: "valid-utf8",
+		Namespace: namespaceName,
 	})
 	s.NoError(err)
 	nsID := descResp.GetNamespaceInfo().GetId()
 
 	delResp, err := s.operatorClient.DeleteNamespace(ctx, &operatorservice.DeleteNamespaceRequest{
-		NamespaceId: nsID,
```

Comment:
So this is a breaking change?

Reply:
I don't know enough about Temporal to answer your question. I believe yes, but there is also this comment from Yimin which was posted internally. 

> Assumption is the setNamespaceIfNotPresent() is a no-op today and can be safely removed. 
> 
> The ask for this task is to verify if we could go with option #3 in the description. If the request already has all Namespace filed set, we can remove this code. 
> We need to write log and emit metrics if the are not set. 

It could imply that this isn't a breaking change from the perspective of someone who uses SDKs.  I also could be misinterpreting, and he is asking to set metrics and logging before making the change. Do you have more context?